history/subscriber/persister.py

```diff
@@ -268,6 +268,35 @@ def on_put(req, resp):
                 'Logging level must be DEBUG, INFO, WARNING, ERROR or CRITICAL!', 'level')
 
 
+def handle_choose_service(config, persister, check):
+
+    messenger = Messenger("Persister", config)
+    messenger.init()
+
+    messenger.create_channel(config.dojot['subjects']['devices'], "r")
+    messenger.create_channel(config.dojot['subjects']['device_data'], "r")
+
+    if conf.dojot_notification_on != True:
+        LOGGER.info("all services")
+
+        # TODO: add notifications to config on dojot-module-python
+        messenger.create_channel("dojot.notifications", "r")
+        messenger.on(config.dojot['subjects']['devices'],
+                     "message", persister.handle_event_devices)
+        messenger.on(config.dojot['subjects']['device_data'],
+                     "message", persister.handle_event_data)
+    else:
+        # Persister Only Notification
+        LOGGER.info(f"One Notification: {conf.dojot_notification_on}")
```

Comment:
The log could be something with "Only notifications are enabled"

Reply:
Ok!